condor-ap/condor_meter

```diff
@@ -189,8 +190,31 @@ def get_num_procs(job_ad):
 
     return procs
 
+
+def become_condor():
```

Comment:
Maybe try [CONDOR_IDS](https://htcondor.readthedocs.io/en/latest/admin-manual/configuration-macros.html#CONDOR_IDS) before the hardcoded 'condor' user.

Reply:
Resolved.